### DISCUSSION TITLE
Default charging profile in case the automation for dynamic charge power fails (unavailable HA)

### DISCUSSION THREAD

**Discussion by LeoAlioth** [lbbrhzn/ocpp#1343] [Q&A]
Hello, 
So, i have successfully integrated EvBox Elvi into HA, and have set up automations to change the charging speed according to grid load and solar.

The problematic part is, that i have no idea what happens in case the HA looses connection to the EVSE.

From what i gather, the profiles can have a time range in which it is valid, and also some sort of a default charging profile.

My end goal would be, for the dynamic charge profile, for which HA determines the allowed power, to only be valid for lets say a minute. And (if i assume correctly), then fall back to the default one (likely set up to minimum charge speeds of 6A) in case HA stops sending updates to the EVSE.

is there a way a could achieve that?

**Comment by mineadle** [comment]
Hi, could you please share any ideas where to dig. I also have a EVBox Elvi and i managed to get Controls, Sensors and Diagnostic data from the Charger. But i cannot control the charger remotely form the HA and cannot create any automations as well. When i switched Charger Control switch manually form the HA it returns back after a second. Thanks!
